client/state/themes/actions/activate-theme.js

```diff
@@ -46,6 +53,12 @@ export function activateTheme(
 					siteId,
 					error,
 				} );
+
+				if ( includes( error.error, 'theme_not_found' ) ) {
```

Comment:
Now that we're moving the code here, I wonder if we can use `String.prototype.includes()` instead of the lodash version.

Reply:
I expected this 🙂 In https://github.com/Automattic/wp-calypso/pull/47084/commits/7ed6a8d49503f6975fda767f3c2616bd0dbce10c, I'm changing the check to exact string match. There's no need to look for substring. Verified by looking at the `POST /sites/:site/themes/mine` backend implementation and the possible errors returned there.

Can you have another look to verify I'm doing the check right?

Reply:
The change looks sensible. Looking at the possible errors, it's the same if we'll check for a substring or an exact match. So exact match is actually an improvement 👍 

Whether the error message is always accurately reflecting the actual reason is a different deal, and it's totally unrelated to this PR.